### ISSUE TITLE
Add lister Bookmarks for Admin>Users

### ISSUE THREAD

**Issue by LostKobrakai** [processwire/processwire-requests#63]
### Short description of the enhancement

Have Bookmarks under Admin>Users like in all other Lister instances

### Why would the enhancement be useful to users?

To have the same conveniencies of saving different filter settings like in standard listers.

Repost of https://github.com/ryancramerdesign/ProcessWire/issues/1997, because it's a pain to manage a lot of users with redoing the filters every time again.

**Comment by dennisspohr** [comment]
Yeah that would be awesome. I would use it for example for filtering different roles, but also by custom fields.

I would already help to make system-pages available on the already existing lister-instance. Then it would be already possible to filter by users.
